index.d.ts

```diff
@@ -0,0 +1,52 @@
+// TypeScript typings for "mocha-testdata@1.1.3".
+
+declare module "mocha-testdata" {
```

Comment:
Well, preventing the types from leaking should be as simple as not exporting them... that's my understanding :wink: